resources/web/management_script/index.php

```diff
@@ -177,17 +177,12 @@
 
 # We start a subprocess so that we can properly log the output
 (
-id=$(security find-certificate -a | awk -F= '/issu/ && /MICROSOFT INTUNE MDM AGENT CA/ { getline; print $2 }')
+id=$(security find-certificate -a | awk -F= '/issu/ && /MICROSOFT INTUNE MDM DEVICE CA/ { getline; print $2 }')
```

Comment:
Hey Simon, have you verified that this works?

When I test this (on macOS Monterey) I receive a prompt:
![image](https://user-images.githubusercontent.com/47246332/220151805-ff6d8b02-969e-42a6-9f8e-36f1c1d597ef.png)

This is due to the fact that not all applications are allowed to use the key of the MDM device cert:
![image](https://user-images.githubusercontent.com/47246332/220152150-1cf2f9ec-a675-4f55-a6fe-d27987a24c43.png)

The MDM agent certificate is created when a script gets pushed from Intune to the macOS device. Access to this key is not restricted:
![image](https://user-images.githubusercontent.com/47246332/220152433-0b86a3e7-f09b-4f6b-b69c-ac03076da8b9.png)

And will therefore not prompt the user with credentials.